db/migrate/20160127193752_create_games.rb

```diff
@@ -0,0 +1,16 @@
+class CreateGames < ActiveRecord::Migration
+  def change
+    create_table :games do |t|
+      t.datetime :start_time, null: false
+      t.belongs_to :week, class_name: 'Week', null: false
```

Comment:
So I googled how to have multiple foreign keys referencing the same table. I came to this [article](http://www.spacevatican.org/2008/5/6/creating-multiple-associations-with-the-same-table/). It's dated back to 2008, so I'm sure there may be a better updated method with Rails 4. 

Pretty much `t.belongs_to :week, class_name: 'Week'` creates the foreign key `week_id` in `Games`.

I'm not sure, but is this doing the same thing that you mentioned in one of your comments.

```
class Game < ActiveRecord::Base
  belongs_to :away_team, class_name: 'Team'
  belongs_to :home_team, class_name: 'Team'
end
```